x/group/spec/state.md

```diff
@@ -0,0 +1,160 @@
+# State & Types
+
+## Groups
+
+Groups are simply aggregations of members.
+
+### ID
+
+```go
+type GroupID []byte
+```
+
+`GroupID` is generated from an auto-incrementing `uint64` preprended with the
+prefix `0`. This prefix allows other group composition mechanisms in the future,
+specifically via token ownership rather than group membership.
+
+### Properties
+
+#### `Members []Member`
+
+An array of `Member` structs:
+
+```go
+// Member specifies a address and a power for a group member
+type Member struct {
+	// The address of a group member. Can be another group or a contract
+	Address sdk.AccAddress `json:"address"`
+	// The integral power of this member with respect to other members and the decision threshold
+	Power sdk.Int `json:"power"`
+}
+```
+
+#### `Owner sdk.ACcAddress`
+
+Owner is the account which "owns" the group and has the permission to 
+add and remove memers.
+
+#### `Memo string`
+
+A single string memo.
+
+#### Indexes
+
+#### `Member`
+
+It should be possible to look-up groups by member address.
+
+#### `Owner`
+
+It should be possible to look-up groups by owner address.
+
+#### `TotalPower sdk.Int`
```

Comment:
Yes, keep this, but this is not an index IMO, rather a calculated field on the group itself.

Reply:
In RDB's that's technically an index... anyway minor point